config/locales/de.devise.yml

```diff
@@ -56,5 +59,5 @@ de:
       not_found: "nicht gefunden"
       not_locked: "ist nicht gesperrt"
       not_saved:
-        one: "Konnte %{resource} nicht speichern: ein Fehler."
-        other: "Konnte %{resource} nicht speichern: %{count} Fehler."
\ No newline at end of file
+        one: "<span style='text-transform:uppercase'>%{resource}</span> konnte nicht gespeichert werden: ein Fehler. Bitte korrigiere deine Eingabe um fortzufahren."
+        other: "<span style='text-transform:uppercase'>%{resource}</span> konnte nicht gespeichert werden: %{count} Fehler. Bitte korrigiere deine Eingaben um fortzufahren."
```

Comment:
Have you tested this in your browser? I think I18n escapes HTML tags by default, in which case this would not work.